types/luxon/src/datetime.d.ts

```diff
@@ -282,6 +282,12 @@ export interface ToISOTimeOptions extends ToISOTimeDurationOptions {
      * @default true
      */
     includeOffset?: boolean | undefined;
+
+    /**
+     * add the time zone format extension
+     * @default false
```

Comment:
I do think they do have an error in docs, the source sets this to false by default,
would be nice if we could submit a bug:
https://moment.github.io/luxon/api-docs/index.html#datetimetoiso